includes/icon-functions.php

```diff
@@ -0,0 +1,78 @@
+<?php
+/**
+ * SVG icons related functions
+ *
+ * @package TenUpScaffold
+ */
+
+/**
+ * An svg template helper.
+ *
+ * Example:
+ * tenupscaffold_svg_icon(
+ *     'instagram',
+ *     array(
+ *         'role' => 'img'
```

Comment:
@colorful-tones can you all all the options in here for the example?

Reply:
Done.